sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -194,8 +194,8 @@ public partial class RequestContext
         public RequestContext() { }
         public System.Threading.CancellationToken CancellationToken { get { throw null; } set { } }
         public Azure.ErrorOptions ErrorOptions { get { throw null; } set { } }
-        public void AddClassifier(Azure.Core.HttpMessageClassifier classifier) { }
-        public void AddClassifier(int statusCode, bool isError) { }
+        public void AddClassificationHandler(Azure.Core.ResponseClassificationHandler classifier) { }
+        public void AddClassificationHandler(int statusCode, bool isError) { }
```

Comment:
It would be better if the name described why one would call it, i.e. change whether responses are considered errors or not , not how we designed it, i.e. you control classification by adding handlers to the classifier.

Reply:
Will fix.

Reply:
How do you feel about `ChangeClassification` ?

Reply:
Actually, I think I'll change it back to `AddClassifier`.  It looks nicer next to AddPolicy.  Let me know if you have concerns about this.  We could do `AddToClassifier` instead if there's a concern with an `AddClassifier` method taking a type that is not a classifier (e.g. a handler).